run_slurm.sb

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash --login
+
+#SBATCH --mail-type=END
+#SBATCH --mail-user=sergii.domanskyi@jax.org
+#SBATCH -p compute
+#SBATCH -q batch
+#SBATCH -t 10:00:00
+#SBATCH --mem=2G
+
+workPath=/fastscratch/domans/work/
+[ ! -d ${workPath} ] && mkdir ${workPath}
+
+~/nextflow run \
+--input=assets/samplesheet_st.csv \
+main.nf \
+-with-timeline -with-trace -with-report \
+-profile slurm,singularity \
+-work-dir $workPath \
+-resume
```

Comment:
I assume this is not part of the workflow and is just a file to submit the pipeline to SLURM?